src/Icicle/Source/ToCore/Prim.hs

```diff
@@ -197,6 +197,15 @@ convertPrim p ann resT xts
   gofun DaysEpoch
    = return $ primmin $ Min.PrimDateTime Min.PrimDateTimeDaysEpoch
 
+  gofun Seq
```

Comment:
trivial, but not obvious. make sure you add a comment saying that ReifyPossibility has already taken care of the sequencing possiblies, or something